package.json

```diff
@@ -152,7 +152,7 @@
     "supertest": "^4.0.2",
     "tough-cookie": "^4.0.0",
     "tsc-watch": "^4.2.8",
-    "typescript": "~4.7.3",
+    "typescript": "~4.8.3",
```

Comment:
We need the TS update for knex v2.4.1? I assume so, just checking.

Reply:
Yes, make it consistent with what knex v2.4.1 is using